src/lib/conditionals/character/Argenti.ts

```diff
@@ -25,46 +29,45 @@ export default (e: Eidolon): CharacterConditional => {
       formItem: 'switch',
       id: 'ultEnhanced',
       name: 'ultEnhanced',
-      text: 'Enhanced ult',
-      title: 'Enhanced ult',
-      content: `Consumes 180 Energy and deals Physical DMG equal to ${precisionRound(ultEnhancedScaling * 100)}% of Argenti's ATK to all enemies,
-      and further deals DMG for 6 extra time(s), with each time dealing Physical DMG equal to ${precisionRound(ultEnhancedExtraHitScaling * 100)}% of Argenti's ATK to a random enemy.`,
+      text: t('Content.0.text'),
+      title: t('Content.0.title'),
+      content: t('Content.0.content', { ultEnhancedExtraHitScaling: TsUtils.precisionRound(100 * ultEnhancedExtraHitScaling), ultEnhancedScaling: TsUtils.precisionRound(100 * ultEnhancedScaling) }),
     },
     {
       formItem: 'switch',
       id: 'enemyHp50',
       name: 'enemyHp50',
-      text: 'Enemy HP ≤ 50% DMG boost',
-      title: 'Courage',
-      content: `Deals 15% more DMG to enemies whose HP percentage is 50% or less.`,
+      text: t('Content.1.text'),
+      title: t('Content.1.title'),
+      content: t('Content.1.content'),
```

Comment:
I feel like these should be tied to the id/name rather than order index. `Argenti.enemyHp50Text` or something. Otherwise switching the order of the content will suddenly mess up the ordering for all the conditionals.

Reply:
I can take this refactor if you're sick of the JSONs by now. Just making sure the reasoning is sound

Reply:
yeah if you could do that that would be great